lib/generators/shopify_app/add_webhook/add_webhook_generator.rb

```diff
@@ -0,0 +1,55 @@
+require 'rails/generators/base'
+
+module ShopifyApp
+  module Generators
+    class AddWebhookGenerator < Rails::Generators::Base
+      source_root File.expand_path('../templates', __FILE__)
+
+      class_option :topic, type: :string, aliases: "-t", required: true
+      class_option :address, type: :string, aliases: "-a", required: true
+
+      def init_webhook_config
+        initializer = load_initializer
+        return if initializer.include?("config.webhooks = [")
```

Comment:
I agree it is a bit fragile but I didn't want to go too far down the rabbit hole for edge cases in the way people define their config.

Maybe what we should do is make this first check a bit less specific i.e. only check for `config.webhooks` here and then warn if we can't find where to inject in the next function.

How does that sound?